src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/DomainEventQueueIntegrationTest.kt

```diff
@@ -36,7 +41,9 @@ class DomainEventQueueIntegrationTest : IntegrationTestBase() {
 
   private fun getNumberOfMessagesCurrentlyOnQueue(hmppsQueue: HmppsQueue): Int? {
     val queueAttributes =
-      hmppsQueue.sqsClient.getQueueAttributes(hmppsQueue.queueUrl, listOf("ApproximateNumberOfMessages"))
-    return queueAttributes.attributes["ApproximateNumberOfMessages"]?.toInt()
+      hmppsQueue.sqsClient.getQueueAttributes(
+        GetQueueAttributesRequest.builder().queueUrl(hmppsQueue.queueUrl).attributeNames(QueueAttributeName.APPROXIMATE_NUMBER_OF_MESSAGES).build()
+      ).get()
```

Comment:
is the `get` because this returns a CompletableFuture or something ?

Reply:
yes, we've switched to an async client so need to do a `get` to resolve